conanfile.py

```diff
@@ -142,8 +137,9 @@ def requirements(self):
             self.requires('protobuf/3.21.12', force=True)
         if self.options.with_postgresql:
             self.requires('libpq/14.5')
+        if (self.options.with_mongodb or self.options.with_kafka):
+            self.requires('cyrus-sasl/2.1.27', force=True)
```

Comment:
i think that it is better when requires goes in together with the target packages
let's distribute the cyrus-sasl requirement back to the mongo's and kafka's modules

Reply:
It will be conflict if 2 components require the same library with different versions, so it's better to declare dependency only once.

Reply:
okey, that makes sense